examples/nni_data_augmentation/README.md

```diff
@@ -0,0 +1,35 @@
+## Tuun data augmentation
+##### Installation
+###### This example
+```shell
+conda create -n augmentation python=3.6 pip -y
```

Comment:
Are we primarily/only using conda here for pytorch installation?

Reply:
Added python venv as alternative.